.github/workflows/main.yml

```diff
@@ -34,8 +34,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [ '1.19', '1.20' ]
-        cassandra_version: [ '4.0.8', '4.1.1' ]
+        go: [ '1.22', '1.23' ]
+        cassandra_version: [ '4.0.13', '4.1.6' ]
```

Comment:
4.1.6 and 4.0.13 are the latest 4.1.x and 4.0.x as of this writing.  But now I'm wondering if we shouldn't be aiming for 4.1.x & 5.0.0 instead.

Trouble is stock ccm has issues with 5.0.0... and supporting a new major version of Cassandra probably needs a ticket (or perhaps even tickets) all to itself.